src/common/Types.ts

```diff
@@ -1121,8 +1123,12 @@ export type SyntheticEvent = {
     readonly cancelable: boolean;
     readonly defaultPrevented: boolean;
     readonly timeStamp: number;
-    readonly currentTarget: EventTarget;
+    readonly currentTarget: EventTarget | NodeHandle;
```

Comment:
I intentionally removed these extra fields from SyntheticEvent because they were not present on all platforms. Do you see a need for them to be present?

Reply:
I thought there were.  They are in the type definitions for [react-native](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/react-native/index.d.ts#L344) and [react](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/react/index.d.ts#L436), and would need to be here to eventually move to `@types/react-native`.

Reply:
I'd prefer to have a ReactXP-specific definition of SyntheticEvent rather than rely on either (or both) of the react and react-native public definition files. The ReactXP version should be a proper subset of all supported platforms.

Reply:
Makes sense.  This change was motivated by my desire to get a dependency on `@types/react-native` rather than using an internal typing file.  This would then be pulled down by consumers of RXP automatically and they could use it whenever diving into RN specific functionality in their app.

Right now, this project has no good solution for this scenario IMO.  I'd like to get valid RN typings for use in a RXP project automatically either by:
1. RXP having a dependency on `@types/react-native`
2. RXP shipping an internal `react-native.d.ts` typing file that gets picked up automatically by consumers.

Alternatively, let people pull down `@types/react-native` themselves, but I'd hope this is something we could solve for them (like is done currently with both `@types/react` and `@types/react-dom`.